### PR TITLE
Fix formatting of README file.

### PR DESCRIPTION
**Pull request by vinayakankugoyal**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Fix the formatting of the README file.
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: images/build/setcap/README.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 
 This image is compiled for multiple architectures.
 
-How to release
+### How to release
+
 If you're editing the Dockerfile or some other thing, please bump the TAG in the Makefile.
 
+```console
 Build and  push images for all the architectures
```
Comment (hasheddan) [inline images/build/setcap/README.md:15]: Would you mind moving this out of code block or making it a comment?

Comment (vinayakankugoyal) [reply images/build/setcap/README.md]: Done! Thanks.

**Review by hasheddan — APPROVED** [review]
/lgtm

**Pull request merged**